lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -44,6 +44,7 @@
     ad_user=dict(type='str', no_log=True),
     password=dict(type='str', no_log=True),
     cloud_environment=dict(type='str'),
+    cert_validation_mode=dict(type='str', choices=['validate', 'ignore'])
```

Comment:
Is there a reason why this can't be a bool option, most modules just have `validate_certs` as a boolean option for this behaviour and it would be nice to be similar.

Reply:
Only reason I do it this way is to leave an escape hatch for other validation behaviors later (eg, "ignore_subject", "ignore_expired")- same way I did for pywinrm.

Reply:
Ok fair enough.